packages/strapi-admin/admin/src/utils/FieldApi.js

```diff
@@ -0,0 +1,36 @@
+import { cloneDeep } from 'lodash';
+import invariant from 'invariant';
+
+class FieldApi {
+  fields = {};
```

Comment:
Since `fields` is a private field, I think we should follow the javascript underscore "convention" for privates fields.

Reply:
I really don't like this convention plus it is not used in any of the back-end files so I think it is ok to leave it this way.